flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBMapState.java

```diff
@@ -520,7 +530,7 @@ private boolean underSameKey(byte[] rawKeyBytes) {
 				return false;
 			}
 
-			for (int i = 0; i < keyPrefixBytes.length; ++i) {
+			for (int i = keyPrefixBytes.length - 1; i >= backend.getKeyGroupPrefixBytes(); --i) {
```

Comment:
nit: maybe `for (int i = keyPrefixBytes.length; --i >= backend.getKeyGroupPrefixBytes();)` looks a bit cleaner.

Reply:
👍 addressed.